Content.Shared/Paper/StampComponent.cs

```diff
@@ -0,0 +1,13 @@
+namespace Content.Shared.Paper
+{
+    [RegisterComponent]
+    public class StampComponent : Component
+    {
+        /// <summary>
+        ///     The name that will be stamped to the piece of paper on examine.
+        /// </summary>
+        [ViewVariables]
+        [DataField("stampedName")]
+        public string StampedName { get; set; } = "A very important person";
```

Comment:
this needs to take in a loc string and not regular text

Reply:
In hindsight it probably doesn't even need to be a pre-filled string, I'll just get rid of it.